runtime/doc/lsp.txt

```diff
@@ -619,19 +619,44 @@ to the callback in the "data" table. The token fields are documented in
 Note: doing anything other than calling
 |vim.lsp.semantic_tokens.highlight_token()| is considered experimental.
 
-Also the following |User| |autocommand|s are provided:
+
+LspRequest                                                        *LspRequest*
+
+After a change to the active set of pending LSP requests. When used from Lua,
```

Comment:
```suggestion
Triggered when a LSP client sends a request to a language server or when the state of a request changes from `pending` to `cancel` or `complete`. When used from Lua,
```

Reply:
If you read the other autocmd docs for Lsp events, they all use this grammar (an assumed "Triggered...") Reading it this way flows better when reading through all the Lsp* autocmd docs.

If you want to reword this one, I should probably reword all of them. What's your preference @mfussenegger ?

Reply:
Could also omit the "Triggered" and start with "When"

My main issue isn't the phrasing/grammar but how vague it is. It doesn't give any information of what kind of changes can happen and at what point in time they happen. One needs to imply it from the example code

Reply:
I rewrote the help entry for this event, let me know if you like this better.